make_prg/from_msa/cluster_sequences.py

```diff
@@ -153,22 +127,6 @@ def extract_clusters(
     return result
 
 
-def merge_sequences(*seqlists: Sequences, first_seq: str) -> ClusteringResult:
-    first_seq_found = False
-    result = list()
-    for seqlist in seqlists:
-        for sequence in seqlist:
-            if sequence == first_seq:
-                first_seq_found = True
-            else:
-                result.append(sequence)
-    if not first_seq_found:
-        raise ValueError(
-            "Provided sequence argument not found in provided list of sequences"
-        )
-    return ClusteringResult(None, [first_seq] + result)
```

Comment:
This was designed to make sure the first sample ID in the input alignment ends up in the first position of the first cluster- is this behaviour still there after removing this?

Reply:
It was not, but was now added back in the last commits. Please recheck